### PR TITLE
Fixing spelling and minor whitespace issues in tests

### PR DESCRIPTION
**Pull request by bschoening**
This PR addresses spelling issues in test code. PyCharm's spell checker was used to flag issues needed correction.

### CODE REVIEW

**Comment by absurdfarce** [comment]
Thanks @bschoening!

Jenkins build kicked off just to confirm there wasn't any regression but based on a quick review of the PR I'm not expecting any.  Assuming we get a reasonably clean build this will go in directly.

**Comment by absurdfarce** [comment]
We did see some test failures that appear to be related to some of these rename operations.  I'll go back through the Jenkins results and see if I can correlate them to changes in this PR later tonight.

**Comment by bschoening** [comment]
I noticed. The pytest seems to have passed, not sure what it means that log scan is failing means and the output doesn't seem to be accessible.

build:passed 
log scan: failing

**Comment by absurdfarce** [comment]
@bschoening We've got a few known flakey tests that complicate determining what's actually new here but I'm pretty sure it can be isolated to the following tests:

```
$ EVENT_LOOP_MANAGER="libev" CASSANDRA_VERSION="4.1.6" pytest -v tests/integration/standard/test_concurrent.py
…
=========================================================================================== short test summary info ============================================================================================
FAILED tests/integration/standard/test_concurrent.py::ClusterTests::test_execute_concurrent - AttributeError: 'ClusterTests' object has no attribute 'execute_concurrent_valiate_tuple'. Did you mean: 'execute_concurrent_validate_tuple'?
FAILED tests/integration/standard/test_concurrent.py::ClusterTests::test_execute_concurrent_with_args - AttributeError: 'ClusterTests' object has no attribute 'execute_concurrent_valiate_tuple'. Did you mean: 'execute_concurrent_validate_tuple'?
FAILED tests/integration/standard/test_concurrent.py::ClusterTests::test_execute_concurrent_with_args_and_execution_profile - AttributeError: 'ClusterTests' object has no attribute 'execute_concurrent_valiate_dict'. Did you mean: 'execute_concurrent_validate_dict'?
FAILED tests/integration/standard/test_concurrent.py::ClusterTests::test_execute_concurrent_with_execution_profile - AttributeError: 'ClusterTests' object has no attribute 'execute_concurrent_valiate_dict'. Did you mean: 'execute_concurrent_validate_dict'?
================================================================================== 4 failed, 7 passed, 224 warnings in 32.66s ==================================================================================
```

```
$ EVENT_LOOP_MANAGER="libev" CASSANDRA_VERSION="4.1.6" pytest -v tests/integration/standard/column_encryption/test_policies.py
…
=========================================================================================== short test summary info ============================================================================================
FAILED tests/integration/standard/column_encryption/test_policies.py::ColumnEncryptionPolicyTest::test_end_to_end_different_cle_contexts_different_ivs - AssertionError: 2 != b'\xb2\xc2\xd2r\xbc\x85g\x96\xef\x04\xb1\[59 chars]\xd7'
FAILED tests/integration/standard/column_encryption/test_policies.py::ColumnEncryptionPolicyTest::test_end_to_end_prepared - AssertionError: 0 != b''
FAILED tests/integration/standard/column_encryption/test_policies.py::ColumnEncryptionPolicyTest::test_end_to_end_simple - AssertionError: 1 != b'\x9f\xd3\xe5m\xc2\x91\x9fd}\r`\xba\xdf\[50 chars]\xfd'
================================================================================== 3 failed, 1 passed, 224 warnings in 43.70s ==================================================================================
```

Both the repro runs above were local runs using your Github branch so _hopefully_ you can repro this locally as well.  If you hit a snag there let me know.

I'm not sure what's going on with the column encryption policy test; it's not clear to me why your changes would've broken that (unless I really missed something).  The ClusterTests failures look to just be knock-on effects from your renaming; those at least should be an easy fix.

File: tests/integration/standard/column_encryption/test_policies.py [hunk 1]
```diff
@@ -34,7 +36,7 @@ def _recreate_keyspace(self, session):
 
     def _create_policy(self, key, iv = None):
         cl_policy = AES256ColumnEncryptionPolicy()
-        col_desc = ColDesc('foo','bar','encrypted')
+        col_desc = ColDesc('foo', ' bar', 'encrypted')
```
Comment (absurdfarce) [inline tests/integration/standard/column_encryption/test_policies.py:39]: @bschoening Went back and looked again and this change is the reason the CLE tests are failing on your branch.  The addition of the extra space here means there's no a mismatch between the actual column name and what's specified in the CLE policy which causes the encryption to behave weirdly.  Undoing just this one change caused the test to pass for me locally.

File: tests/integration/standard/test_concurrent.py [hunk 2]
```diff
@@ -107,11 +107,11 @@ def execute_concurrent_base(self, test_fn, validate_fn, zip_args=True):
                     test_fn(self.session, statement, parameters)
             validate_fn(num_statements, results)
 
-    def execute_concurrent_valiate_tuple(self, num_statements, results):
+    def execute_concurrent_validate_tuple(self, num_statements, results):
             self.assertEqual(num_statements, len(results))
             self.assertEqual([(True, [(i,)]) for i in range(num_statements)], results)
 
-    def execute_concurrent_valiate_dict(self, num_statements, results):
+    def execute_concurrent_validate_dict(self, num_statements, results):
```
Comment (bschoening) [inline tests/integration/standard/test_concurrent.py:114]: I missed 4 other places where 'valiate' was a misspelling

**Comment by absurdfarce** [comment]
@bschoening Anything else you wanted to change on this one?  I re-ran the Jenkins tests after your last round of commits (basically with the repo at 39e92b7f72751b11005bdde360906548d4fee859) and we're in a good spot now.  I'm ready to merge this whenever you say it's done!

**Comment by bschoening** [comment]
@absurdfarce it should be ready.  I was not able to run the integration tests locally, however. Some failures with ccm.  I have a second PR with spelling fixes for the code (this was just tests).

**Pull request merged**